front-end/src/components/MenuBar/index.tsx

```diff
@@ -126,6 +131,7 @@ const MenuBar = ({ className } : Props): JSX.Element => {
 								{loggedOutItems.map((item, index) => <Menu.Item as={NavLink} className='user_items' key={index} {...item} />)}
 							</>
 						}
+						{/* <NetworkDropdown /> */}
```

Comment:
I'd swap the order of the dropdowns. Keeping the user dropdown / login & signup button to the right feels more natural to me.

Reply:
And then it would also need a `margin: auto 1.2rem;`just like the other desktop items.